mantis-publish/mantis-publish-core/src/main/java/io/mantisrx/publish/EventDrainer.java

```diff
@@ -108,7 +110,12 @@ public void run() {
                         streamEventList.clear();
                     }
                 } catch (Exception e) {
-                    LOG.warn("Exception processing events for stream {}", stream, e);
+                    LOG.error("Exception processing events for stream {}", stream, e);
```

Comment:
might be ok to log this at WARN/INFO since there is a metric now to indicate the same.